packages/opentelemetry-exporter-prometheus/src/export/types.ts

```diff
@@ -44,9 +44,8 @@ export interface ExporterConfig {
 
   /**
    * Define if the Prometheus exporter server will be started
-   * @default false
    */
-  startServer?: boolean;
+  startServer: boolean;
```

Comment:
Why did you make this configuration option required? Should it not just default to true?

Reply:
The reason `startServer` defaults to false have been stated at the description of the PR "it grabs a port on the host computer. This type of thing is best left explicit so the user is not surprised" -- which is copy-pasted from previous Gitter chat history I considered as a fair reason (unfortunately Gitter doesn't provide chat history links).

Reply:
I am leaning towards default to true, prometheus is meant to be scraped in most cases. The default port of `9464` is meant for this as per https://github.com/prometheus/prometheus/wiki/Default-port-allocations

Reply:
I believe the gitter chat @legendecas referenced was actually me :)

Think I see this one both ways. It is unfortunate that we have to grab a port, but the exporter really is useless without it so I would say it's probably ok to just do it.